libraries/GCS_MAVLink/GCS_Common.cpp

```diff
@@ -4397,6 +4397,8 @@ void GCS_MAVLINK::send_banner()
     }
 #endif
 
+    send_text(MAV_SEVERITY_INFO, "RCIn: %d", NUM_RC_CHANNELS);
```

Comment:
Let's not spam the GCS with this.  GCS can probably infer from `RC_CHANNELS` if the user should know for some reason.

Reply:
I understand that the RC_CHANNELS message gives the number of radio channels.
I have removed the GCS message.